sky/cli.py

```diff
@@ -2559,17 +2559,12 @@ def autostop(
 
     - The cluster has restarted.
 
-    - An autostop is set when there is no active setting. (Namely, either
-      there's never any autostop setting set, or the previous autostop setting
-      was canceled.) This is useful for restarting the autostop timer.
+    - A new autostop idle time is set.
 
-    Example: say a cluster without any autostop set has been idle for 1 hour,
```

Comment:
Keep/update this scenario?

Reply:
The scenario shoud be similar as the new scenario we have. Do you think it is needed to keep this?

Reply:
I think it'd help the users/readers. Even though the technical impl is the same, it's "setting autostop for the first time" vs. "resetting autostop" scenarios.